pbft/pbft.cpp

```diff
@@ -379,14 +393,16 @@ pbft::handle_preprepare(const pbft_msg& msg, const bzn_envelope& original_msg)
         && std::get<2>(lookup->second.value()) != msg.request_hash())
     {
 
-        LOG(debug) << "Rejecting preprepare because I've already accepted a conflicting one \n";
+        LOG(debug) << "Rejecting preprepare: already accepted a conflicting one";
         return;
     }
     else
     {
         auto op = this->operation_manager->find_or_construct(msg, this->current_peers_ptr());
-        op->record_pbft_msg(msg, original_msg);
-        this->maybe_record_request(msg, op);
+        auto env_copy{original_msg};
+        env_copy.clear_piggybacked_requests();
+        op->record_pbft_msg(msg, env_copy);
+        this->maybe_record_request(msg, original_msg, op);
```

Comment:
So the decision here is to not look at piggybacked requests on prepares/commits/newviews, yes? That is fine; I just want to clarify.

Reply:
I believe that is correct.